platform/jewel/ui/src/test/kotlin/org/jetbrains/jewel/interactions/toggleKey.kt

```diff
@@ -0,0 +1,30 @@
+package org.jetbrains.jewel.interactions
+
+import androidx.compose.ui.input.key.Key
+import androidx.compose.ui.test.ExperimentalTestApi
+import androidx.compose.ui.test.SemanticsNodeInteraction
+import androidx.compose.ui.test.performKeyInput
+
+@OptIn(ExperimentalTestApi::class)
+internal fun SemanticsNodeInteraction.toggleKey(
```

Comment:
Nit: `performKeyPress`? Toggle is usually just changing once, this does down -> wait -> up